.github/workflows/test.yml

```
@@ -1,39 +1,31 @@
 name: test
-
-on:
-  push:
-    branches:
-      - "*"
-  pull_request:
-    branches: [main]
+on: push
 
 jobs:
-  test:
-    name: Test
+  check:
     runs-on: ubuntu-latest
-    container: golang:1.15
+    container: golang:1.16
     steps:
       - uses: actions/checkout@v2
 
-      - name: Download Dependencies
+      - name: Download dependencies
         run: go mod tidy
```

I've changed this in a few places, but we should probably use `go mod download` to avoid running against a different version of the go.mod

```suggestion
        run: go mod download
```